image_provisioner/inventory/hosts

```diff
@@ -0,0 +1,55 @@
+# vim:ft=ansible:
+localhost ansible_connection=local ansible_python_interpreter=/usr/bin/python
+
+[all:vars]
+# AWS vars
+aws_access_key=
+aws_secret_key=
+region=us-west-2
+key_name=
+group_id=
+instance_type=m4.xlarge
+vpc_subnet_id=
+base_image=
+# repo install
+rhel_int_repo_hostname=
+# os kickstart
+ssh_repo=
+# collectd install
+config_type=ose
+collectd_interval=10
+collectd_status=started
+graphite_host=
+graphite_prefix=ose
+# clone repos
+github_token=
+# pbench config
+pbench_internal_hostname=pbench\..*com$
+pbench_alternate_hostname=
+# docker config
+# sda2 for openstack, sdb/xvdb for ec2
+config_repo=
+docker_login=no
+docker_storage_device=/dev/xvdb
```

Comment:
Running the setup-image.yaml using this inventory - it could not find xvdb on the resize second partition step.   If I use vda2 it works.   Am I doing something wrong with the base image?  or should be running a different inventory for setup-image.yaml?